cmd/kubeadm/app/cmd/config_test.go

```diff
@@ -41,7 +44,7 @@ import (
 )
 
 const (
-	defaultNumberOfImages = 8
+	defaultNumberOfImages = 7
```

Comment:
why change this?

Reply:
@thank you for the review.

> why change this?

I think I explained it in the commit message:
```
default number of control plane images is 7:
  - k8s.gcr.io/kube-apiserver
  - k8s.gcr.io/kube-controller-manager
  - k8s.gcr.io/kube-scheduler
  - k8s.gcr.io/kube-proxy
  - k8s.gcr.io/pause
  - k8s.gcr.io/etcd
  - k8s.gcr.io/coredns

defaultNumberOfImages constant was set to 8 incorrectly to
count empty line in the output produced by
'kubeadm config images list' command.
```

In short - 8 is not a default number of images as you may think. It's a number of lines in the text output. If I keep it as it was it would influence tests cases for JSON/YAML output.